app/assets/javascripts/costs/editinplace.js

```diff
@@ -46,19 +46,23 @@ function edit(obj, name, obj_value) {
   var obj_value = typeof(obj_value) != 'undefined' ? obj_value : obj.innerHTML;
   var parsed = getCurrencyValue(obj_value);
   var value = parsed[0];
-  var currency = parsed[1]
+  var currency = parsed[1];
 
-  var span_in = '<span id="'+obj.id+'_editor">'
-    var text = '<input id="'+obj.id+'_edit" name="'+name+'" size="7" value="'+value+'" class="currency" /> '+currency;
-    var button = '<input id="'+obj.id+'_cancel" type="image" '+ _cancelButtonAttributes  +' /> ';
-  var span_end = '</span>';
+  var button = '<span id="'+obj.id+'_cancel" class="form--field-affix -transparent"><input id="'+obj.id+'_cancel" type="image" '+ _cancelButtonAttributes  +' /> </span>';
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>'_cancelButtonAttributes' is not defined.